MAUI/Scheduler/cell-selection.md

```diff
@@ -76,7 +76,7 @@ this.Scheduler.CellSelectionView.CornerRadius = 2;
 
 The thickness of the `Stroke` of Scheduler can be customized by using the `StrokeWidth` property in the `CellSelectionView`.  
 
-Note: The `StrokeWidth` is only applicable for `Stroke` property.
+N> The thickness of the selection view `Stroke` can be customized by using the `StrokeWidth` property in the `CellSelectionView`.
```

Comment:
The `StrokeWidth` is only applicable for the `Stroke` property.

Reply:
Updated